content/_changelogs/10.5.0.md

```diff
@@ -0,0 +1,88 @@
+  [#23089](https://github.com/cypress-io/cypress/issues/23089) &
+  [#22819](https://github.com/cypress-io/cypress/issues/22819) &
+  [#23096](https://github.com/cypress-io/cypress/issues/23096) &
+  [#23000](https://github.com/cypress-io/cypress/issues/23000) &
+  [#22991](https://github.com/cypress-io/cypress/issues/22991) &
+  [#23125](https://github.com/cypress-io/cypress/issues/23125).
+
+**Bugfixes:**
+
+- Fixes an issue where usage of the --spec option would result in no specs found
+  if the absolute project path included glob pattern characters. Fixes
+  [#22272](https://github.com/cypress-io/cypress/issues/22272).
+- Webpack entry points are no longer preserved for CT Webpack projects. Fixes
+  [#23224](https://github.com/cypress-io/cypress/pull/23224)
+- Fixes an issue where session state was not being reset when changing between
+  specs in open mode. Fixes
+  [#23146](https://github.com/cypress-io/cypress/pull/23146)
+- Add delay to header tooltips so they don't pop open unless hovered on. Fixes
+  [#23259](https://github.com/cypress-io/cypress/pull/23259)
+- Fixed an edge-case bug where Cypress would hang in run-mode if an error was
+  through from a test:before:run event handler. Fixes
+  [#23039](https://github.com/cypress-io/cypress/pull/23039).
+- Fixes an issue introduced in 10.3.0 where network logs could be missing
+  certain information or cause certain requests to be delayed by 500ms. Fixes
+  [#23227](https://github.com/cypress-io/cypress/pull/23227).
+- In the 'Create Blank Spec' dialog, pressing enter or return when the focus is
+  on the spec path input field will create the blank spec. Fixes
+  [#21815](https://github.com/cypress-io/cypress/pull/21815).
+- Fixes `lastMountedReactDom.unmountComponentAtNode is not a function` error
+  when using React 18 mount. Fixes
+  [#23081](https://github.com/cypress-io/cypress/pull/23081).
+- Cypress will remember the spec filter in the 'Search Specs' field during your
+  session and when a project is reopened. The same filter will be used in the
+  main specs list as well as the inline specs list present in the Test Runner.
+  Fixes [#21837](https://github.com/cypress-io/cypress/pull/21837).
+- Fixed an issue that could lead to infinite recursion and thus a crash when
+  running tests that make use of `cy.intercept`. Fixes
+  [#22693](https://github.com/cypress-io/cypress/pull/22693).
+- In the Launchpad, the project's name will no longer link to the 'Choose
+  Testing Type' step of the launchpad. This behavior was removed because in many
+  situations it is not valid to go back to the testing type picker and in the
+  situation where it is valid, there's already a 'Switch testing types' link on
+  the same page. Fixes
+  [#21911](https://github.com/cypress-io/cypress/pull/21911).
+- CT projects using Vite v3 work with Node versions >=17. Fixes
+  [#23042](https://github.com/cypress-io/cypress/pull/23042) &
+  [#23114](https://github.com/cypress-io/cypress/pull/23114).
```

Comment:
I think these just redirect to issues?

Reply:
Updated [b24000b](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/b24000bfc44224a88960308c6ca23b16ca9f2a23)